source/CommonJobs/CommonJobs.Application.EvalForm/Commands/GetEvaluationCalifications.cs

```diff
@@ -94,7 +94,8 @@ private CalificationsDto GetEvaluatorEvaluation(CalificationsEvaluationDto evalu
             {
                 View = UserView.Evaluation,
                 Evaluation = evaluationDto,
-                Califications = califications.Where(c => _loggedUser == c.EvaluatorEmployee || c.Owner == CalificationType.Auto).ToList()
+                Califications = califications.Where
+                (c => _loggedUser == c.EvaluatorEmployee || c.Owner == CalificationType.Auto || (evaluationDto.ReadyForDevolution && c.Owner == CalificationType.Company)).ToList()
```

Comment:
I understand what you mean. But I think I failed to explain the scenario correctly. The califications IEnumerable has only the califications for this particular evaluation. Not every calification in the period. 

So, for example, if employee A has been evaluated by responsible B and evaluator C, then this list will have the auto calification (A to A), the responsible calification (B to A), the logged evaluator calification (C to A) and the company calification (Company to A).

So, those conditions will get: A to A, C to A and Company to A.

It's like a subset of califications, only the ones that belong to the specific evaluation being loaded.